README.Rmd

```diff
@@ -0,0 +1,40 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
```

Comment:
This should not be here?

Reply:
Oh, I was confused by the "please edit that file" which is referring to THIS file